test/foundry/CollateralAgreement.t.sol

```diff
@@ -249,4 +210,170 @@ contract CollateralAgreementTest is AgreementFrameworkTestBase {
 
         _aliceExpectsErrorWhenDisputing(agreementId, "AgreementIsFinalized()");
     }
+
+    /* ====================================================================== //
+                              DISPUTE SETTLEMENT TESTS
+    // ====================================================================== */
+
+    function _setupDispute() internal returns (bytes32 agreementId) {
+        agreementId = _createAgreement();
+
+        _bobJoinsAgreement(agreementId);
+        _aliceJoinsAgreementWithPermit(agreementId);
+
+        hevm.prank(bob);
+        framework.disputeAgreement(agreementId);
+    }
+
+    function _getValidSettlement() internal view returns (PositionParams[] memory) {
+        PositionParams[] memory settlement = new PositionParams[](3);
+
+        settlement[0] = PositionParams(bob, 1.5 * 1e18);
+        settlement[1] = PositionParams(alice, 0);
+        settlement[2] = PositionParams(address(0xD011), 1.5 * 1e18);
```

Comment:
Maybe add a username for this address as well, to follow the "arbitrator"/"bob"/"alice" pattern?